### PR TITLE
WIP: Welcome Mexican documentation v13.0

### PR DESCRIPTION
**Pull request by nhomar**

### CODE REVIEW

File: accounting/localizations/mexico.rst [hunk 1]
```diff
@@ -2,233 +2,347 @@
...
 
-.. image:: media/mexico01.png
+.. image:: media/mexico03.png
    :align: center
 
-.. tip::
-   When creating a database from www.odoo.com, if you choose Mexico
-   as country when creating your account, the mexican localization will be
-   automatically installed.
+In the form for the company set the proper information for your mexican company (Remember you must to set Mexico as country
```
Comment (jga-odoo) [inline accounting/localizations/mexico.rst]:
```suggestion
Set the proper information for your Mexican company in the company information form. (Remember you must to set Mexico as the country 
```

File: accounting/localizations/mexico.rst [hunk 2]
```diff
@@ -2,233 +2,347 @@
...
-.. image:: media/mexico01.png
+.. image:: media/mexico03.png
    :align: center
 
-.. tip::
-   When creating a database from www.odoo.com, if you choose Mexico
-   as country when creating your account, the mexican localization will be
-   automatically installed.
+In the form for the company set the proper information for your mexican company (Remember you must to set Mexico as country 
+in your company in order to allow all to work properly following mexican workflow),
```
Comment (jga-odoo) [inline accounting/localizations/mexico.rst]:
```suggestion
in your company in order to allow everything to work properly with regards to Mexican workflows).
```

File: accounting/localizations/mexico.rst [hunk 3]
```diff
@@ -2,233 +2,347 @@
...
+.. image:: media/mexico03.png
    :align: center
 
-.. tip::
-   When creating a database from www.odoo.com, if you choose Mexico
-   as country when creating your account, the mexican localization will be
-   automatically installed.
+In the form for the company set the proper information for your mexican company (Remember you must to set Mexico as country 
+in your company in order to allow all to work properly following mexican workflow), 
+**Please be sure to not add a new country Mexico by mistake, select the one from the list which is pre created**.
```
Comment (jga-odoo) [inline accounting/localizations/mexico.rst]:
```suggestion
**Please make sure you do NOT create a new country called "Mexico" by mistake; always select the one from the list of pre-defined countries**.
```

File: accounting/localizations/mexico.rst [hunk 4]
```diff
@@ -2,233 +2,347 @@
...
----------------------------------------------
+Set the proper Fiscal Regime
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-To enable this requirement in Mexico go to configuration in accounting Go in
-:menuselection:`Accounting --> Settings` and enable the option on the image
-with this you will be able to generate the signed invoice (CFDI 3.2 and 3.3)
-and generate the payment complement signed as well (3.3 only) all fully
-integrate with the normal invoicing flow in Odoo.
+Go in :menuselection:`Accounting --> Settings` and enter a valid Fiscal Regime.
```
Comment (jga-odoo) [inline accounting/localizations/mexico.rst]:
```suggestion
Go to :menuselection:`Accounting --> Settings` and enter a valid Fiscal Regime.
```

File: accounting/localizations/mexico.rst [hunk 5]
```diff
@@ -2,233 +2,347 @@
...
 
-.. image:: media/mexico02.png
+.. image:: media/mexico04.png
    :align: center
 
-.. _mx-legal-info:
+.. tip::
+   For testing environment you must set **General de Ley Personas Morales**
```
Comment (jga-odoo) [inline accounting/localizations/mexico.rst]:
```suggestion
   In testing environments you must set **General de Ley Personas Morales**
```

File: accounting/localizations/mexico.rst [hunk 6]
```diff
@@ -2,233 +2,348 @@
...
+.. image:: media/mexico61.png
+   :align: center
+
+.. image:: media/mexico62.png
    :align: center
 
+**b.** Payment first and then reconcile with invoices.
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+TODO: CONTINUE THIS
```
Comment (ren-odoo) [inline accounting/localizations/mexico.rst:329]: @nhomar I think that we should only specify that this option uses the standard process.

File: accounting/localizations/mexico.rst [hunk 7]
```diff
@@ -257,74 +372,41 @@ If the invoice is not paid.
...
 
-   `According to the SAT documentation`_ a payment is classified as ``PUE`` if
-   the invoice was agreed to be fully payed before the 17th of the next
-   calendar month (the next month of the CFDI date), any other condition
-   will generate a ``PPD`` invoice.
+Credit Notes
+------------
 
-   **1.2. How can I get this with Odoo?**
+# todo
```
Comment (ren-odoo) [inline accounting/localizations/mexico.rst:383]: @nhomar we can remove this todo an  add it on new PR once the documentation for Credit Notes is ready.

File: accounting/localizations/mexico.rst [hunk 8]
```diff
@@ -257,74 +372,41 @@ If the invoice is not paid.
...
 
-   In order to set the appropriate CFDI payment term (PPD or PUE), you can
-   easily set it by using the ``Payment Terms`` defined in the invoice.
+Debit Notes
+------------
 
-   - If an invoice is generated without ``Payment Term`` the attribute
-     ``MetodoPago`` will be ``PUE``.
+# todo
```
Comment (ren-odoo) [inline accounting/localizations/mexico.rst:388]: @nhomar we can remove this todo an add it on new PR once the documentation for Debit s is ready.

File: accounting/localizations/mexico.rst [hunk 9]
```diff
@@ -257,74 +372,41 @@ If the invoice is not paid.
...
-   - Today, if an invoice is generated with ``Payment Term`` ``30 Net Days``
-     and the ``Due Date`` is higher than the day 17 of the next month the
-     ``MetodoPago`` will be ``PPD``.
+Payment in advance
+-------------------
 
-   - If having a ``Payment Term`` with 2 lines or more, for example
-     ``30% Advance End of Following Month``, this is an installments term,
-     then the attribute ``MetodoPago`` will be ``PPD``.
+# todo
```
Comment (ren-odoo) [inline accounting/localizations/mexico.rst:393]: @nhomar we can remove this todo an add it on new PR once the documentation for Advance Payments is ready

**Comment by ren-odoo** [comment]
Hi @nhomar @luistorresm 

Can you help us to integrate the requested changes and feedback from @jcgarza-github and to cleanup the #todos, is better to ingrate those todos on a future PR so we can have merged everything that is ready for now. 

Let us know if you any questions.   Thank you in advance!

**Comment by jcgarza-github** [comment]
Thank you @ren-odoo -- Please note that the github account I use for work is @jga-odoo  :)

**Comment by ren-odoo** [comment]
Hi @nhomar did you have time to clean the to-dos and add the corrections from @jga-odoo? Do you have an ETA?

Thank you.

**Comment by jcs-odoo** [comment]
Hello all + @vbe-odoo and @ren-odoo ,

What would be the status of this Pull Request, now that changes were made to 13.0 and 14.0?
By the way, if this PR were to be updated, now, be aware that some image files aren't named the same way anymore in 13.0, and that the content in 14.0 is totally different, now :)

Cheers :)
Jonathan

**Comment by jga-odoo** [comment]
Hello @jcs-odoo : I think that after the recent update to the PPD/PUE date calculation (thank you!) the rest can probably remain as is--not ideal, but also probably not worth the effort (unless @ren-odoo has other thoughts!)

**Comment by vbe-odoo** [comment]
@jcs-odoo Hi! We can cancel/close this PR as the documentation done is more updated.
We already checked this with @ren-odoo and she agreed on this.

Many many thanks @jga-odoo !
Bests!

**Comment by jcs-odoo** [comment]
@jga-odoo @vbe-odoo 

As you prefer, really!
Then, I close it now but if you change your mind and think it'd be better to do it anyway, we can reopen the pull request and update the commit so it works.

Let me know if you change your mind and need assistance to update all this.

Cheers,
Jonathan

**Comment by vbe-odoo** [comment]
Hi @jcs-odoo !
Thanks for the update!
We will let you know if there's anything to change so we reopen the PR.
Thanks again! =D
cc' @ren-odoo @jga-odoo

**Pull request closed without merging**